website/docs/r/jwt_auth_backend_role.html.md

```diff
@@ -73,6 +73,10 @@ The following arguments are supported:
 * `bound_claims` - (Optional) If set, a map of claims/values to match against.
   The expected value may be a single string or a list of strings.
 
+* `bound_claims_type` - (Optional) How to interpret values in the claims/values
+  map (`bound_claims`): can be either `string` (exact match) or `glob` (wildcard
+  match). Requires Vault 1.4.0 or above. Defaults to `string`.
```

Comment:
I think since we're leaving the default up to Vault, we don't necessarily need to mention the default here.
```suggestion
  match). Requires Vault 1.4.0 or above.
```